src/cli/commands/cache.js

```diff
@@ -55,7 +55,7 @@ export const {run, setFlags, examples} = buildSubCommands('cache', {
   },
 
   dir(config: Config, reporter: Reporter) {
-    reporter.log(config.cacheFolder);
+    reporter.log(config.cacheFolder, {force: true});
```

Comment:
I am not happy with this solution since I think it needs a better overall solution as we have discussed. Let's have a `TODO` comment here saying "Don't use force option and fix this ASAP" or something along those lines so we don't forget?